apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx

```diff
@@ -51,16 +52,17 @@ const SurveyResultsHeader = React.createClass({
     this.setState({
       selectedCourse: course,
       filteredWorkshops: filteredWorkshops,
-      selectedWorkshopId: firstWorkshopId
+      selectedWorkshopId: firstWorkshopId,
     });
 
-    this.setSurveyPanel(firstWorkshopId);
+    this.setSurveyPanel(firstWorkshopId, this.getWorkshopFriendlyName(filteredWorkshops[0]));
```

Comment:
After other discussions, scratch this. They want a different (shorter) format here.